iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/LoadDetailImpl.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.impl.extensions;
+
+import com.powsybl.iidm.network.Load;
+import com.powsybl.iidm.network.extensions.LoadDetail;
+import com.powsybl.iidm.network.impl.AbstractMultiVariantIdentifiableExtension;
+import gnu.trove.list.array.TDoubleArrayList;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public class LoadDetailImpl extends AbstractMultiVariantIdentifiableExtension<Load> implements LoadDetail {
+
+    private final TDoubleArrayList fixedActivePower;
+
+    private final TDoubleArrayList fixedReactivePower;
+
+    private final TDoubleArrayList variableActivePower;
+
+    private final TDoubleArrayList variableReactivePower;
```

Comment:
You could use `TFloatArrayList` instead. Or shouldn't we rather use a ArrayList of POJOs containing the four power values? That would allow a simple solution for variants with undefined LoadDetail, by using a null POJO.

Reply:
I used `TFloatArrayList` to be consistent with other similar equipments/implementations